maturity-level-two/tests/Codit.IntegrationTest/CarTest.cs

```diff
@@ -0,0 +1,63 @@
+using Xunit;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.AspNetCore.Hosting;
+using System.Net.Http;
+using System.Net;
+using System.Threading.Tasks;
+using FluentAssertions;
+
+using Codit.LevelTwo;
+
+namespace Codit.IntegrationTest
+{
+    public class CarTest
+    {
+        private readonly HttpClient _httpClient;
+        public CarTest()
+        {
+            if (_httpClient != null) return;
+            var srv = new TestServer(new WebHostBuilder()
+                .UseEnvironment("Development")
+                .UseStartup<Startup>());
+
+            _httpClient = srv.CreateClient();
```

Comment:
For each testcase that is executed, you're creating a new HttpClient instance.

It would be better to have it like this:
```csharp
private static readonly HttpClient Http;

static CarTest()
{
   var srv = new TestServer(.....);

   Http = srv.CreateClient();
}
```